### PR TITLE
Fix Indonesian translation for several countries.

### PR DESCRIPTION
**Pull request by vanestorz**
## What does this PR do?

Fix Indonesian translation for several countries to match translation from wikipedia.

### CODE REVIEW

File: app/config/locale/translations/id.countries.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 return [
-    'AF' => 'Afganistan',
+    'AF' => 'Afghanistan',
```
Comment (spielers) [inline app/config/locale/translations/id.countries.php:4]: According to my understanding, Afghanistan  => Afganistan   refer to [this ](https://www.101languages.net/indonesian/country-names-indonesian/)

File: app/config/locale/translations/id.countries.php [hunk 2]
```diff
@@ -165,11 +165,11 @@
     'SK' => 'Slovakia',
     'SI' => 'Slovenia',
     'SE' => 'Swedia',
-    'SZ' => 'Swaziland',
```
Comment (spielers) [inline app/config/locale/translations/id.countries.php:168]: Swaziland  ==> Swaziland   refer to [this](https://www.101languages.net/indonesian/country-names-indonesian/)

**Review by spielers — CHANGES_REQUESTED** [review]
Hey @vanestorz You can refer to [this ](https://www.101languages.net/indonesian/country-names-indonesian/)to Know more 👍 

please do the required changes

best,
spielers

File: app/config/locale/translations/id.countries.php (hunk 1 above)
Comment (vanestorz) [reply app/config/locale/translations/id.countries.php:4]: i misread the title on wikipedia sorry.

File: app/config/locale/translations/id.countries.php (hunk 2 above)
Comment (vanestorz) [reply app/config/locale/translations/id.countries.php:168]: I refer to https://id.wikipedia.org/wiki/Eswatini and https://en.wikipedia.org/wiki/Eswatini, should it being updated? or keep it as it is?

Comment (vanestorz) [reply app/config/locale/translations/id.countries.php:168]: > Swaziland ==> Swaziland refer to [this](https://www.101languages.net/indonesian/country-names-indonesian/)

I refer to [Wikipedia: Indonesian](https://id.wikipedia.org/wiki/Eswatini) and [Wikipedia: English](https://en.wikipedia.org/wiki/Eswatini), it is officialy renamed in 2018, also the SZ code if i refer to [ISO](https://www.iso.org/obp/ui/#iso:code:3166:SZ) it is representing Eswatini. Should it being updated? or keep it as it is?

**Comment by spielers** [comment]
hey @vanestorz  please resolve the conflict 

Best
speielrs

**Comment by vanestorz** [comment]
> hey @vanestorz please resolve the conflict
> 
> Best
> speielrs

Done, i also updated the translation for DR Congo.

**Comment by spielers** [comment]
Will have a look
Thanks 👍

**Comment by eldadfux** [comment]
@spielers good for merge?

**Comment by spielers** [comment]
Good to go @eldadfux

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**